app/models/eligibility_verification.rb

```diff
@@ -17,24 +30,38 @@ def self.expiration_warning
   end
 
   def will_expire_within_warning_period?
+    return if expiration_date.blank?
```

Comment:
I know it's functionally the same, but I prefer predicate methods to always return `true` or `false`